### DISCUSSION TITLE
Writing Custom Metadata

### DISCUSSION THREAD

**Discussion by echai58** [delta-io/delta-rs#1977] [Q&A]
Hi, I see that I am able to write custom metadata via the configuration dict on creation of a delta table. It looks like there is no ability to update the configuration dict after creation, though - is this intended? Regardless of custom metadata, I feel this would make sense be able to modify, for updating things like retention, etc. For custom metadata, it would be immensely useful to be able to update custom metadata fields as part of the configuration dict.

**Comment by ion-elgreco** [comment]
@echai58 this will be addressed with this PR: https://github.com/delta-io/delta-rs/pull/1994

**Reply by echai58** [reply]
Amazing, thanks. I see that this is merged and released as 0.15.0 earlier today. When will this version be available via conda?
